modules/shared/components/atoms/avatar/index.tsx

```diff
@@ -0,0 +1,83 @@
+import type { FC, ReactElement } from 'react';
+import className from 'classnames';
+import React from 'react';
+import Image from 'next/image';
+import type { IAvatar } from './IAvatar';
+
+export const determineImageSize = (componentSize: string): number => {
+  const smallSize = 32;
+  const mediumSize = 40;
+  const largeSize = 56;
+  if (componentSize === 'small') {
+    return smallSize;
+  }
+  if (componentSize === 'medium') {
+    return mediumSize;
+  }
+  return largeSize;
+};
+
+const Index: FC<IAvatar.IProps> = ({ size, variant, imgSrc }): ReactElement => {
+  const classes = className({
+    'w-8 h-8': size === 'small',
```

Comment:
The Avatar component. takes `size` as props. Let us consider if the developer who used the component typed `Small` instead of `small` or there is a typo in the word ? `no warning but the code will not work as expected `. 

- using  [string-literal-types](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-1-8.html#string-literal-types) will prevent such errors and when you use the component it will show in vs code the possible values that can be used

Reply:
Should I use enum also or using string-literal-types is good enough?